src/Search/SingleHackerView.tsx

```diff
@@ -0,0 +1,208 @@
+import * as React from 'react';
+
+import { Box, Flex } from '@rebass/grid';
+import { toast } from 'react-toastify';
+
+import { Hacker } from '../api';
+import { HackerStatus, IAccount, IHacker } from '../config';
+import { Button, H1, H2, Paragraph } from '../shared/Elements';
+import ViewPDFComponent from '../shared/Elements/ViewPDF';
+import { Form, StyledSelect } from '../shared/Form';
+import theme from '../shared/Styles/theme';
+import { getOptionsFromEnum } from '../util';
+
+interface IHackerViewProps {
+  hacker: IHacker;
+}
+
+interface IHackerViewState {
+  canEdit: boolean;
+  isLoading: boolean;
+  status: HackerStatus;
+}
+
+class SingleHackerView extends React.Component<
+  IHackerViewProps,
+  IHackerViewState
+> {
+  constructor(props: IHackerViewProps) {
+    super(props);
+    this.state = {
+      status: HackerStatus.HACKER_STATUS_APPLIED,
+      canEdit: true,
+      isLoading: false,
+    };
+  }
+
+  public componentDidMount() {
+    const {
+      hacker: { status },
+    } = this.props;
+    this.setState({ status });
+    this.submit = this.submit.bind(this);
+  }
+
+  public render() {
+    const { hacker } = this.props;
+    const account = (hacker.accountId as IAccount) || {};
+    return (
+      <article style={{ maxWidth: '800px', margin: '0 auto' }}>
```

Comment:
Should we make a `styled-component` out of this?

Reply:
just refactored the styles out lol